src/platform/primitives.ios.ts

```diff
@@ -7,5 +7,3 @@ export const styled = {
   Text: styles.Text,
   View: styles.View,
 }
-
-export const styledWrapper = styles
```

Comment:
@ds300 - Realized I didn't need this export because styled-components offers an `.extends` function!